server/internal/web/server.go

```diff
@@ -6,13 +6,15 @@ import (
 	"github.com/rs/zerolog"
 
 	"github.com/oxidrive/oxidrive/server/internal/core"
+	"github.com/oxidrive/oxidrive/server/internal/web/handler"
 )
 
 type Config struct {
 	Address        string
 	Application    *core.Application
 	Logger         zerolog.Logger
 	FrontendFolder string
+	handler.Config
```

Comment:
I am not sure about the name `handler.Config`

Reply:
Why not?

Reply:
Maybe it's more server config, rather than handler config?

Reply:
It creates multiple configs scattered in different places, and it may be hard to have a quick look at them